explainer.md

```diff
@@ -415,17 +415,21 @@ xrSession.addEventListener('inputsourceschange', (ev) => {
 });
 ```
 
+The properties of an XRInputSource object are immutable. If a device can be manipulated in such a way that these properties can change, the `XRInputSource` will be removed and recreated.
+
 ### Input poses
 
-Each input source can queried a `XRInputPose` using the `getInputPose()` function of any `XRPresentationFrame`. Getting the pose requires passing in the `XRInputSource` you want the pose for, as well as the `XRCoordinateSystem` the pose values should be given in, just like `getDevicePose()`. Similar to `getDevicePose()` the requested pose may return `null` in cases where tracking has been lost.
+Each input source can query a `XRInputPose` using the `getInputPose()` function of any `XRPresentationFrame`. Getting the pose requires passing in the `XRInputSource` you want the pose for, as well as the `XRCoordinateSystem` the pose values should be given in, just like `getDevicePose()`. `getInputPose()` may return `null` in cases where tracking has been lost (similar to `getDevicePose()`), or the given `XRInputSource` instance is no longer connected or available.
+
+If an input source can be tracked the `XRInputPose`'s `gripMatrix` will indicate the device's position and orientation. If only position or orientation is trackable (not both), the `gripMatrix` will return a transform matrix applying only the trackable pose. An example of this case is for physical hands on some AR devices, that only have a tracked position. The `gripMatrix` will be `null` if the input source isn't trackable. 
```

Comment:
"If only position or orientation is trackable (not both), the `gripMatrix` will return a transform matrix applying only the trackable pose"

The wording on this is a little weird to me. I assume you mean that, for example, if the input source can only track position the transform matrix will not contain any rotation component? Not sure how to state that more clearly though.

Also, does this imply that the transform matrix should not apply an arm model for 3DoF controllers? Because if so I'm pretty strongly opposed to that.

Reply:
Your interpretation is correct, but I'll have a think about wording to make this a bit clearer. However, it was not the intention to disallow arm modeling for 3DoF controllers; would changing "will return" -> "may return" be sufficient?

Reply:
Will return -> may return is good enough for me, especially since this change does preserve the explicit mention of arm models later on.